x-pack/platform/plugins/shared/onechat/server/services/runner/model_provider.ts

```diff
@@ -58,7 +58,11 @@ export const createModelProvider = ({
     const chatModel = await inference.getChatModel({
       request,
       connectorId,
-      chatModelOptions: {},
+      chatModelOptions: {
+        telemetryMetadata: {
```

Comment:
Given that we needed to set this in 2 places can we refactor this into a plugin-level constant sth like `export const MODEL_TELEMETRY_METADATA = {pluginId: one_chat}` 

So that if we ever need to make changes to this payload we just need to update it in a single place

Reply:
Fair, I thought it'd be too early for that kind of abstraction but I don't have a strong enough opinion against it - addressed it in [7789240](https://github.com/elastic/kibana/pull/231258/commits/7789240d2f620f7e43dadecf02134bbca3010d3c)